### DISCUSSION TITLE
问题记录：toggleterm 版本提示；dashboard 无法识别

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#71] [General]
# toggleterm 问题
启动nvim会出现提示
vim.highlight.create is deprecated, use vim.api.nvim_set_hl instead. See :h deprecated
解决：`use({ "akinsho/toggleterm.nvim", branch = 'main' })` toggleter 插件指定分支

# dashboard 无法识别
之前使用的是这样的配置，无法被识别到
![image](https://user-images.githubusercontent.com/77786952/182281316-be4365dc-53f9-4a2a-97dc-1cc7e9e02f71.png)

参考dashboard 的 GitHub，使用了 lua 语言的配置，可以显示，但是很多命令就要重新配置了。
问题：
1. 为什么dashboard 在 MacOS 中工作正常，但是在 WSL 中就不太正常？

**Comment by kang8** [comment]
[dashboard-nvim](https://github.com/glepnir/dashboard-nvim) 前不久[重构](https://github.com/glepnir/dashboard-nvim/issues/139)了一次，根据 REDEME 重新配置一下试试。

**Reply by Linya-IronMan** [reply]
哦哦哦 ~~ 谢谢
